FindTown/FindTown/Presentation/MapScene/Map/SubViews/MapStoreCollectionViewCell.swift

```diff
@@ -122,6 +164,10 @@ private extension MapStoreCollectionViewCell {
         contentView.layer.borderWidth = 1
         contentView.layer.cornerRadius = 16
         contentView.layer.masksToBounds = true
+        
+        contentView.layer.shadowRadius = 10
+        contentView.layer.shadowOpacity = 1
+        contentView.layer.shadowColor = FindTownColor.grey4.color.cgColor
```

Comment:
```suggestion
      contentView.layer.addCustomShadow(shadowX: 0, shadowY: 2, shadowColor: FindTownColor.black.color.withAlphaComponent(0.4), blur: 10.0, spread: 0, alpha: 0.8)
```

셀 그림자 값들이 town cell 그림자 값에서 alpha만 다르더라구요! 요렇게 사용하시면 될 것 같습니다..! :)

Reply:
오!!! 이거 색을 정확하게 몰라서 헤맸는데 감사합니다 :)